cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsPathResolver.scala

```diff
@@ -135,6 +150,7 @@ object MarthaResponseSupport {
       name = fileName,
       gsUri = gcsUrl,
       googleServiceAccount = response.googleServiceAccount,
+      fileName = None,
```

Comment:
Why not extract the `name` key from Martha V2 response [here](https://github.com/broadinstitute/cromwell/blob/222976d88c53d451c7d68fae46ffada9451e6da2/cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsPathResolver.scala#L73-L76) and pass it along as `fileName` here?